documentation/Mod_Template/manifest.json

```diff
@@ -18,9 +18,6 @@
             "background_music":
             [
                 "music/track_1.mp3",
-                "music/track_2.mp3",
```

Comment:
Well, we think that having a minimum of four different sound tracks in you GX mod, so that there is unique and noticable music separation, and that having four different sound tracks can actually help your GX mod to appel more stylish and more separated from other GX mods makingmit more preferable, but I guess it's your choise to have one track...